linkerd/http-metrics/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "linkerd2-http-metrics"
+version = "0.1.0"
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
+edition = "2018"
+publish = false
+
+[dependencies]
+bytes = "0.4"
+futures = "0.1"
+h2 = "0.1"
+http = "0.1"
+http-body = "0.1"
+httparse = "1.2"
```

Comment:
where do we use `httparse` in this crate? AFAICT, we only use it in [version.rs](https://github.com/linkerd/linkerd2-proxy/blob/8c8134800b614820df75ba47632987b9f663faeb/linkerd/proxy/http/src/version.rs) which doesn't seem to have moved in this change?